src/gemdat/volume.py

```diff
@@ -288,6 +288,26 @@ def to_structure(
 
         return structure
 
+    def get_free_energy(
+        self,
+        kBT,
+    ) -> np.ndarray:
+        """Estimate the free energy from volume.
+
+        Parameters
+        ----------
+        kBT : float
+            The temperature of the thermostat
```

Comment:
What is thermostat?

Reply:
It is more understandable if we say 'temperature of the simulation', so I have made the change.

However, the temperature fluctuates along the trajectory and the value that we use is the one that the simulation is trying to impose (the thermostat).